src/instruments/src/Common/hooks.tsx

```diff
@@ -60,3 +60,28 @@ export const useInteractionEvents = (events: string[], handler: (any?) => void):
         ...events,
     ]);
 };
+
+export const useSessionStorage = (key: string, initialValue: any) => {
```

Comment:
Why not use Redux?

Reply:
@Benjozork any update?

Reply:
@hiaaryan I raised looking at React useReducer and useContext hooks as an alternative to Redux. I did raise it on the dev support channel on Discord. This is the only pending thing for the moment. I was considering doing the work for it today in a separate branch to discuss. Any thoughts on this before I do are more than welcome.

Reply:
`useContext` has done allocations that are usually bad for GC in coherent so we would like to avoid that

Although local/session storage is slow in Coherent.

Let's just go with `useContext`.